Bottom: 10e471da2bc2b79bd0ba8e7673b758dbc3ad1ecc
Top:    14d8fa0c88ba389671896d3602dd733bc60ef608
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-05 22:28:00 +0000

Refresh of extend-board-export

---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index ea52c65..42232a0 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -309,13 +309,14 @@ object3d_from_contours (const POLYAREA *contours,
           double x1, y1;
 
           /* Update which contour we're looking at */
-          if (offset_in_ct == ct_npoints) {
-            offset_in_ct = 0;
-            ct = ct->next;
-            ct_npoints = get_contour_npoints (ct);
-          }
+          if (offset_in_ct == ct_npoints)
+            {
+              offset_in_ct = 0;
+              ct = ct->next;
+              ct_npoints = get_contour_npoints (ct);
+            }
 
-              get_contour_coord_n_in_step_mm (ct, offset_in_ct, &x1, &y1);
+          get_contour_coord_n_in_step_mm (ct, offset_in_ct, &x1, &y1);
 
           vertices[i]           = make_vertex3d (x1, y1, COORD_TO_STEP_Z (PCB, zbot)); /* Bottom */
           vertices[npoints + i] = make_vertex3d (x1, y1, COORD_TO_STEP_Z (PCB, ztop)); /* Top */
